src/Services/CodeTypesService.php

```diff
@@ -368,4 +368,17 @@ public function lookupFromValueset($code, $codeType, $codeSystem)
         );
         return $value;
     }
+
+    public function dischargeOptionIdFromCode($formatted_code)
+    {
+        $listService = new ListService();
+        $ret = $listService->getOptionsByListName('discharge-disposition', ['codes' => $formatted_code]) ?? '';
+        return $ret[0]['option_id'];
+    }
+
+    public function dischargeCodeFromOptionId($option_id)
+    {
+        $listService = new ListService();
```

Comment:
@kchapple this is where you'll get the discharge code to populate the
QDM Attribute: Discharge Disposition sdtc:dischargeDispositionCode. 
I still have work to populate form_encounter with code instead of option_id if not in list plus add all the codes to the list. ugh
It's populated in encounter:
![image](https://user-images.githubusercontent.com/1859985/160969298-85a26c25-bd7e-4104-80ea-3da17c93a2cd.png)